components/Ripple/src/MDCRippleTouchController.h

```diff
@@ -107,4 +135,19 @@
          didProcessRippleView:(nonnull MDCRippleView *)rippleView
               atTouchLocation:(CGPoint)location;
 
+/**
+ Inserts the ripple view into the given view.
```

Comment:
This documentation doesn't seem quite right. I believe this method is the opportunity for the delegate to perform the insertion (or not).  It doesn't actually insert the the ripple.

Reply:
Done